src/views/ModelPlan/TaskList/Basics/Milestones/index.scss

```diff
@@ -0,0 +1,7 @@
+.milestone-form {
+  .fieldGroup__wrapper {
+    display: flex;
+    grid-gap: units(3);
+    align-items: flex-end;
+  }
```

Comment:
Using util classes had weird alignment issues on the FE when there is an error. See images below
|   |   |
|:--:|---|
|with util classes|<img width="548" alt="Screen Shot 2022-05-04 at 4 24 50 PM" src="https://user-images.githubusercontent.com/2349702/167665234-35152062-e219-4e8f-b627-037c2b5de210.png">|
|with sass styles|<img width="542" alt="Screen Shot 2022-05-04 at 4 25 09 PM" src="https://user-images.githubusercontent.com/2349702/167665309-86c850a0-fd8d-4fe6-8faf-75916e9bad33.png">|